jeffrey/make_custom_table.sql

```diff
@@ -0,0 +1,16 @@
+create table jeffreys_table (
+    name varchar(500),
+    favorite_food varchar(30) CHECK (
+        favorite_food in (
+            'pizza',
+            'ramen',
+            'baguette',
+            'beans and toast'
+        )
+    )
```

Comment:
For further reading, see Postgres' official documentation on `CHECK` constraints!
https://www.postgresql.org/docs/current/ddl-constraints.html#DDL-CONSTRAINTS-CHECK-CONSTRAINTS